### DISCUSSION TITLE
Exporting Users?

### DISCUSSION THREAD

**Discussion by TwistedMexi** [Glench/ExtPay#166] [Q&A]
Is there a way to export users should I choose to migrate to another payment system in the future? I don't see any such controls on the dashboard. I'm reluctant to release with ExtensionPay considering the PayPal support doesn't seem to be getting implemented any time soon, and if I'm locked into this platform permanently, that would not be ideal.

Thanks.

**Comment by Glench** [comment]
Of course! Just email us whenever you'd like to migrate off.
